redisvl/extensions/llmcache/semantic.py

```diff
@@ -248,6 +276,7 @@ def check(
         vector: Optional[List[float]] = None,
         num_results: int = 1,
         return_fields: Optional[List[str]] = None,
+        tags: Optional[Union[List[str], str]] = None,
```

Comment:
I like the idea here of allowing for these tags to filter. Wonder if we could lean on the more generic concept and allow user to pass a `FilterExpression` object which then gets applied to the `RangeQuery` or `VectorQuery`? This would align with the notion of letting users customize their filters broadly and use the redisvl tools to shape the filter at run time!

Reply:
I like that idea, especially since we already have good support and examples of how to construct filters.